backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -387,6 +409,10 @@ def set_user_infos(self, context, request: TracimRequest, hapic_data=None):
         Set user info data
         """
         app_config = request.registry.settings["CFG"]  # type: CFG
+        try:
+            check_has_write_rights(UserReadOnlyFields.PUBLIC_NAME, app_config, request)
+        except ReadOnlyFieldException:
```

Comment:
why do you try/except in this case? I would expect using a ` @hapic.handle_exception(ReadOnlyFieldException, HTTPStatus.BAD_REQUEST)`

Is it a miss? If not, then an explanation in a comment is a good idea, I think

Reply:
@lebouquetin Since `public_name` is always sent for this request (even when changing language) it would always throw an error. Instead we make sure the `public_name` doesn't change but we do not prevent other changes. I will put a comment

Reply:
Added the comment